modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* You installed the required packages and performed the necessary configuration
+on the machine that you run the playbook on.
+* You prepared the RHEL hosts for installation.
+
+.Procedure
+
+Perform the following steps on the machine that you prepared to run the
+playbook:
+
+. Extract the pull secret for your cluster.
++
+----
+$ oc -n openshift-config get -o jsonpath='{.data.\.dockerconfigjson}' secret pull-secret | base64 -d | jq .
```

Comment:
to get info, need to login the cluster or provide config file thru
1. flag --config
2. the KUBECONFIG environment variable
3. In your home directory as ~/.kube/config

Reply:
Ok. I'm adding this to the module about preparing a machine to run the playbook:
```
* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`,
on the machine that you run the playbook on.
* Log in as a user with `cluster-admin` permission.
```